File: engine/engine-model/src/main/scala/tools/packet/foreigneffect/CancelChair.scala
package tools.packet.foreigneffect

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class CancelChair(private var _itemId: Int) extends PacketInput {
  def itemId: Int = _itemId

  override def opcode(): SendOpcode = SendOpcode.CANCEL_CHAIR
}

File: engine/engine-model/src/main/java/tools/packet/foreigneffect/CancelChair.java
package tools.packet.foreigneffect;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record CancelChair(Integer itemId) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.CANCEL_CHAIR;
   }
}